api/proto/ankaios.proto

```diff
@@ -155,15 +162,29 @@ message UpdateWorkloadState {
   repeated WorkloadState workloadStates = 1; /// A list of workload states.
 }
 
+message Response {
```

Comment:
I think the idea was to have the two most important messages, `ToServer` and `FromServer` at the beginning of the file. The remaining messages are sorted by their first usage (mostly, some are sorted incorrect but `Response` and `Request` have the correct location).

I would prefer to keep `ToServer` and `FromServer` at the beginning. These are the messages you most of the time interested in and for the other message your IDE/Editor should help you.

Reply:
When refactoring the examples and working with the proto file like an external developer would do, I noticed that because of this ordering I was scrolling and searching a lot which was sometimes not nice (because even if you are searching in the ide, a view terms are reused as variable names and so you must change your search option and all that stuff to find the correct term without pressing a lot of enter until you reach the search term). But we can keep. It will get shorter if we split the proto file into internal and external messages.